search/src/main/kotlin/com/ritense/search/service/SearchFieldV2Service.kt

```diff
@@ -62,4 +62,10 @@ class SearchFieldV2Service(
             this?.let { searchFieldV2Repository.delete(it) }
         }
 
+    fun updateList(ownerId: String, searchFieldV2: List<SearchFieldV2>): List<SearchFieldV2> {
+        var order = 0
```

Comment:
This can be replaced by `.mapIndexed` combined with the `.copy` method

Reply:
fixed